internal/cli/internal/command/machine/list.go

```diff
@@ -55,54 +56,37 @@ func runMachineList(ctx context.Context) (err error) {
 	var (
 		appName = app.NameFromContext(ctx)
 		client  = client.FromContext(ctx).API()
-		stream  = iostreams.FromContext(ctx)
-		cfg     = config.FromContext(ctx)
+		io      = iostreams.FromContext(ctx)
 	)
 
-	state := flag.GetString(ctx, "state")
-	if flag.GetBool(ctx, "all") {
-		state = ""
+	if appName == "" {
+		return fmt.Errorf("app is not found")
 	}
-
-	machines, err := client.ListMachines(ctx, appName, state)
+	app, err := client.GetApp(ctx, appName)
 	if err != nil {
-		return fmt.Errorf("could not get list of machines: %w", err)
+		return err
 	}
-
-	if flag.GetBool(ctx, "quiet") {
-		for _, machine := range machines {
-			fmt.Fprintf(stream.Out, "%s\n", machine.ID)
-		}
-		return nil
+	flapsClient, err := flaps.New(ctx, app)
+	if err != nil {
+		return fmt.Errorf("list of machines could not be retrieved: %w", err)
 	}
 
-	if cfg.JSONOutput {
-		return render.JSON(stream.Out, machines)
+	machines, err := flapsClient.Get(ctx, "")
+	if err != nil {
+		return fmt.Errorf("machines could not be retrieved")
 	}
 
-	rows := [][]string{}
-
-	for _, machine := range machines {
-		var ipv6 string
-
-		for _, ip := range machine.IPs.Nodes {
-			if ip.Family == "v6" && ip.Kind == "privatenet" {
-				ipv6 = ip.IP
-			}
-		}
-
-		rows = append(rows, []string{
-			machine.ID,
-			machine.Config.Image,
-			machine.CreatedAt.String(),
-			machine.State,
-			machine.Region,
-			machine.Name,
-			ipv6,
-		})
+	var listOfMachines []api.V1Machine
+	if err = json.Unmarshal(machines, &listOfMachines); err != nil {
+		return fmt.Errorf("list of machines could not be retrieved")
+	}
 
+	fmt.Fprintf(io.Out, "%d machines have been retrieved\n\n", len(listOfMachines))
+	for _, machine := range listOfMachines {
+		fmt.Fprintf(io.Out, " Machine ID: %s\n", machine.ID)
+		fmt.Fprintf(io.Out, " Instance ID: %s\n", machine.InstanceID)
+		fmt.Fprintf(io.Out, " State: %s\n\n", machine.State)
 	}
-	_ = render.Table(stream.Out, appName, rows, "ID", "Image", "Created", "State", "Region", "Name", "IP Address")
```

Comment:
I think it’d be good to keep the same output format, given some apps could have a lot of machines, keeping the output for a machine on one line helps with readability. The response from list machines _should_ still contain enough information to populate the same columns and if not we can fix.